r2i/tensorflow/engine.h

```diff
@@ -49,6 +49,11 @@ class Engine : public IEngine {
 
   std::shared_ptr<TF_Session> session;
   std::shared_ptr<Model> model;
+
```

Comment:
I prefer the old way of handling the prediction. Indeed I wish other backends followed this approach.

Reply:
I've reverted this change.